### ISSUE TITLE
Can't start app

### ISSUE THREAD

**Issue by elenat82** [felisio/ionic-es6-webpack#6]
Hi,

sorry but I am pretty new to Gulp, Webpack etc.....

I followed your instructions:
- git clone of project
- cd into created project
- npm install
- gulp

At this point on my console I shoud see the Access urls but instead nothing appears and if I go to localhost:3000 or localhost:3000/home I can't find my app.

This is the output after gulp:
![after gulp](https://user-images.githubusercontent.com/12762367/29064887-b869428e-7c2a-11e7-88dd-df0c269bfcfb.PNG)

**Comment by elenat82** [comment]
update:
I opened another console and I typed ionic serve, that is what happened:
![update1](https://user-images.githubusercontent.com/12762367/29065723-4253fc26-7c2d-11e7-8d5d-9c028bf46841.PNG)

Now I can see my app at http://localhost:8100/#!/app/home but nothing works because I have this error in the dev tools console: "WARNING: Tried to load angular more than once." at bundle.js:80142
![devtools](https://user-images.githubusercontent.com/12762367/29065814-9aa563c4-7c2d-11e7-9636-a646a6ff7baa.PNG)

**Comment by elenat82** [comment]
update 2:
now I uncommented some things in the head of index.html and now it looks like this:
`    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
    <link href="css/style.css" rel="stylesheet">

    
    <link href="css/ionic.app.min.css" rel="stylesheet">
    
     <script src="lib/ionic/js/ionic.bundle.js"></script>

    
    <script src="cordova.js"></script>

    
     <script src="app/app.js"></script> 
    
     <script src="bundle.js"></script>
`

Now the app appears but still I get the errors:
app.js:1 Uncaught SyntaxError: Unexpected token import 
bundle.js:33773 WARNING: Tried to load angular more than once.
bundle.js:80142 WARNING: Tried to load angular more than once.

And then if I reload the page I get several errors and I need to comment / uncomment again the links one at a time until I get the app back.
